### ISSUE TITLE
Freezing on startup (reduced window; no taskbar icon) on Ubuntu-MATE

### ISSUE THREAD

**Issue by adam-funk** [laurent22/joplin#8231]
Platform: Ubuntu-MATE 22.04.2 LTS
Joplin version: Joplin-2.10.19.AppImage with +x flag set installed in $HOME/bin

I installed this yesterday on my laptop, then on a phone and a tablet, then set up a Joplin cloud account for syncing. Now when I launch Joplin on the laptop (either from a custom launcher on a toolbar or the command line with `bin/Joplin-2.10.19.AppImage`, it opens taking up about 3/4 of the desktop, with  a white area taking up almost all of the desktop, as [reported by someone else on discourse in 2021](https://discourse.joplinapp.org/t/joplin-freezing-on-startup-linux-mint-21-mate/29717). It doesn't put a button in the taskbar and is very difficult to kill (by typing "blind" in the terminal covered up by the white area and the seized-up Joplin app).

The command-line does give this error or warning: `xdg-settings: invalid application name`

I have now enabled debugging but after using Ctrl-C and restarting the app, it keeps working. I'll try again when I log out and back in again later.

**Comment by adam-funk** [comment]
![Screenshot at 2023-05-23 08-33-07](https://github.com/laurent22/joplin/assets/25798054/cafa4954-6653-4813-a607-7d23527c5e9d)

Here's a screenshot from this morning's crash.

**Comment by xavierfontaine** [comment]
Same here!

I encounter this issue on Ubuntu Mate 21.10.
* It started happening after updating Joplin to `2.10.19` from an earlier version (I don't know the version number, but there were no 'external links' available at that time.) 
* The same issue occurs with pre-release `2.11.4`.

If I run the AppImage from the command line, the stdout/stderr is as follows:
```
xdg-settings: invalid application name
Try 'xdg-settings --help' for more information.
mesa: for the --simplifycfg-sink-common option: may only occur zero or one times!
mesa: for the --global-isel-abort option: may only occur zero or one times!
mesa: for the --amdgpu-atomic-optimizations option: may only occur zero or one times!
```

**Comment by adam-funk** [comment]
I've tested a few more things and the launch behavior depends on how Joplin was last closed/killed, **not** how it was just launched.

- If I used "quit" within Joplin or closed the window, it always jams up next time.
- If I launched from the terminal and used Ctrl-C, it always (I think) works next time.
- If I used `killall Joplin-2.10.19.AppImage @joplinapp-desktop`, it sometimes works next time but usually jams up.

Hope that helps.

**Comment by lmcquellin** [comment]
I am also observing similar behaviour on Ubuntu-MATE 22.04.2 LTS using Joplin 2.10.19, installed via the install script. First launch, everything was fine. Set up synconisation target to a file system (PCloud Drive) and quit via the file menu. Launched at a later time resulted in the behaviour described by @adam-funk first comment and later screenshot. Jolplin was unresponsive and did not appear on the taskbar. Killed processes using `killall Joplin-2.10.19.AppImage @joplinapp-desktop` (two monitors made this easier).

I figured my synchronisation settings were the cause of my issues so I deleted the contents of .config/joplin-desktop and relauched. Joplin still launched in the 3/4 screen mode without a taskbar icon but was responsive to mouse inputs. It appeared to behave as if it was in full screen mode, as I had to hover over areas where the menu bar would be to access different menus. I quit through the file menu.

From then on I have observed inconsistent behaviour on launch using File > Quit when exiting the previous session. Sometimes it is fine, sometimes it operates in that 3/4 screen mode (not jammed up, but not functioning as intended).

**Comment by lmcquellin** [comment]
My current fix for Mate 22.04 is to enable "show tray icon" which means I can quit, even if the UI does not behave as expected. Usually after a few quits and re-launches, it works and I just keep the session open for the remainder of my day.
